client/signup/steps/plans/index.jsx

```diff
@@ -120,6 +127,10 @@ export class PlansStep extends Component {
 			return intervalType;
 		}
 
+		if ( this.state.experiment?.variationName !== null ) {
```

Comment:
We should also check for experiment slug here. Event though `this.state.experiment` only stores the experiment we are running at the moment which is`'calypso_signup_monthly_plans_default_202201_v1'`, it's confusing when one spots this section of the code and will require them to do extra context digging.

Reply:
Good point, I've added this check.